styleguide/components/31-molecules/menu/_menu.scss

```diff
@@ -3,16 +3,21 @@ $hamburger-menu-width: 320px;
 nav.menu {
   margin: 0;
 
+  @include tablet {
+    flex-basis: 100%;
+    flex-grow: 1;
+    order: 9999;
```

Comment:
Does this work in IE11?

Reply:
Yes :), but maybe that's because it's rendered as the last element. This order property is just something optional :-)